src/engine/Input/Pointer.ts

```diff
@@ -121,13 +121,8 @@ export class Pointers extends Class {
    /**
     * Initializes pointer event listeners
     */
-   public init(scope: PointerScope = PointerScope.Document): void {
-      var target = <any>document;
-      if (scope === PointerScope.Document) {
-         target = document;
-      } else {
-         target = <any>this._engine.canvas;
-      }
+   public init(target?: any): void {
```

Comment:
It may make sense to type target as `GlobalEventHandlers` which Document and Element both implement.